examples/cloudplatform/projectiambinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: ProjectIAMBinding
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudplatform/v1beta1/projectiambinding
+    upjet.upbound.io/manual-intervention: "Depends on a valid project & member"
```

Comment:
```suggestion
```
Could you please remove manual-intervention, these parameters are passed by the uptest?